Update faq.md (#103637)
Azure/azure-content: articles/kubernetes-fleet/faq.md

Azure Kubernetes Fleet Manager resource is a regional resource. Support for region failover for disaster recovery use cases is in the [roadmap](https://aka.ms/fleet/roadmap).

## What happens when the user changes the cluster identity of a joined cluster?

> The member agent will automatically be aware of the identity and will communicate with the hub using the new token.  \

Reviewer: ```suggestion
The member agent will automatically pick up the new identity and communicate with the hub using the new token.  \
```

However, the fleet hub is unaware of the new identity, and needs to be reconfigured to authorize the member. \

> To do so, you can perform a PutMember operation to mitigate this issue.

Reviewer: ```suggestion
You can join the member cluster to the fleet again by performing a PutMember operation to notify the fleet hub to pick up the new identity.
```
